scripts/audio-processing/extract_wav_from_mp4_mp.py

```diff
@@ -0,0 +1,51 @@
+# AudioExtract.py
+import glob
+
+import moviepy.editor as me
+import multiprocessing as mp
+from pathlib import Path
+from time import time
+
+overwrite = False
+
+
+def save_audio(f):
+    try:
+        clip = me.VideoFileClip(f[0])
+        clip.audio.write_audiofile(f[1])
+
+    except OSError:
+        print(f)
+
+
+# Step 1: Init multiprocessing.Pool()
```

Comment:
Check out https://joblib.readthedocs.io/en/latest/, has nice convenicne funcs for doing this kinda stuff, sklearn uses it

Reply:
oh i heard of this. I will check out and add a part of save actions

Reply:
also, don't mind all the commented code. Next commits of these files that will be gone